baselines/compute/samples/generated/v1/snippet_metadata.addresses.v1.json.baseline

```diff
@@ -0,0 +1,243 @@
+{
+  "clientLibrary": {
+    "name": "nodejs-compute",
+    "version": "0.1.0",
+    "language": "TYPESCRIPT",
+    "apis": [
+      {
+        "id": "Addresses",
+        "version": "v1"
+      },
+      {
+        "id": "RegionOperations",
+        "version": "v1"
+      }
+    ]
+  },
+  "snippets": [
+    {
+      "regionTag": "compute_v1_generated_Addresses_AggregatedList_async",
+      "title": "Addresses aggregatedList Sample",
+      "origin": "API_DEFINITION",
+      "description": " Retrieves an aggregated list of addresses.",
+      "canonical": true,
+      "file": "addresses.aggregated_list.js",
+      "language": "JAVASCRIPT",
+      "segments": [
+        {
+          "start": 20,
+          "end": 72,
+          "type": "FULL"
+        }
+      ],
+      "clientMethod": {
+        "shortName": "AggregatedList",
+        "fullName": "google.cloud.compute.v1.AggregatedList",
```

Comment:
Wondering if this should be:

```suggestion
        "fullName": "google.cloud.compute.v1.AddressesClient.AggregatedList",
```

Reply:
I think the service is actually: 
```
              {
                "service": "google.cloud.compute.v1.AcceleratorTypes",
                "method": "AggregatedList"
              }
```

But I think I get what you're saying, it should be: 
`"google.cloud.compute.v1.AcceleratorTypes.AggregatedList"`